packages/integration_test/lib/integration_test.dart

```diff
@@ -2,6 +2,20 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// Tool for self-driving testing of Flutter code on devices and emulators.
+///
+/// It adapts `flutter_test` results into a format that is compatible with
+/// `flutter drive` and native Android instrumentation testing.
+///
+/// To use, follow the [*Integration testing* guide](https://flutter.dev/docs/testing/integration-tests).
```

Comment:
```suggestion
/// To use, follow the [Integration testing guide](https://flutter.dev/docs/testing/integration-tests).
```
I would suggest removing the italics `*` from the "Integration testing" text, as it doesn't look right when generating the documentation:

![image](https://user-images.githubusercontent.com/2494376/144458969-8e99f34b-94b0-4027-9022-80c4f64d3583.png)